pb/Makefile

```diff
@@ -0,0 +1,12 @@
+PB = $(wildcard *.proto)
+GO = $(PB:.proto=.pb.go)
+
+all: $(GO)
+
+%.pb.go: %.proto
+		protoc --proto_path=$(GOPATH)/src:. --gogofast_out=. $<
```

Comment:
If possible switch to `gogofaster` from `gogofast`. The only drawback is that in case of optional fields they can't be differentiated between set and default values.

Reply:
Done too.